### ISSUE TITLE
Conflicts with fontawesome

### ISSUE THREAD

**Issue by carc1n0gen** [erikflowers/weather-icons#182]
It looks like this pack conflicts with fontawesome.  Is there a version that doesn't?

**Comment by erikflowers** [comment]
I am not aware of any Font Awesome conflicts, it was designed to specifically not conflict with FA at all.

Can you elaborate?

**Comment by BHoma91** [comment]
+1

Having the same issue when running with webpack and ES6.
In webpack.config.js, were exporting the module with the following loaders
`{
            test: /\.(woff|ttf|eot|svg)(\?v=[a-z0-9]\.[a-z0-9]\.[a-z0-9])?$/,
            loader: "url-loader?limit=10000&mimetype=application/font-woff"
        }, {
            test: /\.woff2(\?v=\d+\.\d+\.\d+)?$/,
            loader: "url-loader?limit=10000&mimetype=application/font-woff"
        }, {
            test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/,
            loader: "url-loader?limit=10000&mimetype=application/octet-stream"
        }, {
            test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
            loader: "url-loader?limit=10000&mimetype=image/svg+xml"
        }`
In App.js
`import 'font-awesome/css/font-awesome.css';
import 'weather-icons/css/weather-icons.css';`

If I remove the code from webpack.config.js font-awesome won't load and weather-icons will.
Other wise font-awesome will load and weather-icons will not render.

**Comment by gammons** [comment]
Font Awesome is also using private use area, and in the same unicode locations.  For instance, icon `wi-day-sunny-overcast` maps to unicode character `f00c`.  But, `f00c` is also defined in font awesome [as the check character](http://fontawesome.io/icon/check/).  When you use both FA and weather icons, there is indeed a conflict.

**Comment by erikflowers** [comment]
This is true that they use the same space, but since they use a different
“font-family” they don’t conflict. It’s just like any other font, they all
use the same unicode addresses. I have used Font Awesome and Weather Icons
together.



On October 19, 2017 at 3:52:58 AM, Grant Ammons (notifications@github.com)
wrote:

Font Awesome is also using private use area, and in the same unicode
locations. For instance, icon wi-day-sunny-overcast maps to unicode
character f00c. But, f00c is also defined in font awesome as the check
character <http://fontawesome.io/icon/check/>. When you use both FA and
weather icons, there is indeed a conflict.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/erikflowers/weather-icons/issues/182#issuecomment-337872391>,
or mute the thread

**Comment by erikflowers** [comment]
They're even used on the same page here: https://erikflowers.github.io/weather-icons/

**Comment by carc1n0gen** [comment]
I believe my issue was that I was using an online tool called fontello, which lets you pick and choose font icons from various packs, and builds a single font face.  That is probably why I experienced overlap

**Comment by gammons** [comment]
that makes sense.  my particular use case is that I'm setting up a little weather thingy in [polybar](https://github.com/jaagr/polybar) and I simply cannot get the fonts to cooperate.  Font awesome always seems to override other fonts, butI can't seem to get to the bottom of why.

Anyways, I blamed the wrong culprit :flushed: sorry about that!
